src/openrpc/appcatalog.json

```diff
@@ -0,0 +1,369 @@
+{
+  "openrpc": "1.2.4",
+  "info": {
+    "title": "AppCatalog",
```

Comment:
I wonder if we still want this API, since discovery-sdk was never released. Maybe we need to revisit this before we add it here.

Reply:
Roger, I'll drop it.

Want me to drop the entire `/discovery/` directory, or just this module?